udata_gouvfr/views.py

```diff
@@ -46,6 +50,44 @@ def redevances():
     return theme.render('redevances.html')
 
 
+@cache.cached(50)
+def get_page_content(slug):
+    repo = current_app.config.get('PAGES_GH_REPO_NAME')
```

Comment:
perhaps we could also add something like : 
`PAGES_GH_REPO_NAME = ''`
in the `udata/settings.py` file, just as a reminder ?

Reply:
Good question. I think we should avoid polluting udata's config with stuff specific to gouvfr. It will be easier to deploy (no udata PR/release needed) and to maintain. Right now we don't have a settings.py for gouvfr but it's possible for an extension https://github.com/opendatateam/udata-tabular-preview/blob/master/udata_tabular_preview/settings.py.

Reply:
Done now. I added the banner settings too, so @JulienParis can you make a PR to remove them from udata? 🙏

Reply:
Ok. I also note we'll have to update the settings in the infra repo

Reply:
No, I don't think so. Everything comes from `udata.cfg`, so as long as the name is the same it's ok.